src/pages/snake.astro

```diff
@@ -0,0 +1,222 @@
+---
+import Layout from '../layouts/Layout.astro';
+---
+
+<Layout title="Gamejam 2024 | Snake" />
+
+<script type="module" is:inline>
+  const CELL_SIZE = 30;
+  const FRAME_RATE = 10;
+
+  const snake = {
+    body: [],
+    dx: 1,
+    dy: 0
+  };
+
+  const food = {
+    x: Math.floor(Math.random() * CELL_SIZE),
+    y: Math.floor(Math.random() * CELL_SIZE)
```

Comment:
Generate the food coordinates after canvas is created so that it doesn't spawn off-screen (especially in mobile).

(or set the initial food coordinates as a constant like (4, 4), that could do the trick)